lib/central/account/queries/user_queries.ex

```diff
@@ -77,6 +77,11 @@ defmodule Central.Account.UserQueries do
       where: users.email == ^email
   end
 
+  def _search(query, :email_lower, value) do
```

Comment:
copied this pattern from the user search where there is a "name" and a "name_lower" query available

https://github.com/beyond-all-reason/teiserver/blob/4c73f45609db6bc93aecbc18990559ab93c6c0e4/lib/central/account/queries/user_queries.ex#L70